lerna-package-lock.json

```diff
@@ -1904,6 +1904,275 @@
 				"tslib": "^1.10.0"
 			}
 		},
+		"@fluid-experimental/get-container": {
+			"version": "0.57.2",
```

Comment:
Looks like you might need to run `npm i` from root again to fix up the lerna-package-lock.